perun-openapi/openapi.yml

```
@@ -4859,6 +4894,179 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  #################################################
+  #                                               #
+  # AuditMessagesManager                          #
+  #                                               #
+  #################################################
+
+  /json/AuditMessagesManager/getMessages:
+    get:
+      tags:
+        - AuditMessagesManager
+      operationId: getMessages
+      summary: |
+        Returns 100 newest audit messages from audit log. If there is a less messages than 100,
+        then all of them are returned OR Returns exact number of newest audit messages defined by
+        'count' param (disregarding message IDs). If there is less messages present, then all of them are returned..
+      parameters:
+        - name: count
+          in: query
+          schema:
+            type: integer
+          description: Messages limit
+          required: false
```

Please use the short syntax to save lines:
```suggestion
        - { name: count, in: query, schema: { type: integer },  description: "Messages limit",  required: false }
```